docs/assets/storage_provider.md

```diff
@@ -65,6 +65,18 @@ By default, the GCS client use the credentials setup up on the machine.
 
 If `GOOGLE_APPLICATION_CREDENTIALS` is provided, it should point to a local JSON service account file, which we use to instantiate the client with `google.cloud.storage.Client.from_service_account_json`
 
+### Using Azure blob storage
+
+Use `STORAGE_PROVIDER=az` to connect to Azure blob storage.
```

Comment:
s/STORAGE_PROVIDER/MODELKIT_STORAGE_PROVIDER/ :) (we should update the other env variables too 👍)

Reply:
done